osquery/sql/virtual_table.cpp

```diff
@@ -848,19 +848,24 @@ static int xBestIndex(sqlite3_vtab* tab, sqlite3_index_info* pIdxInfo) {
   }
 
   UsedColumns colsUsed;
-  if (pIdxInfo->colUsed > 0) {
+  UsedColumnsBitset colsUsedBitset(pIdxInfo->colUsed);
+  if (colsUsedBitset.any()) {
     for (size_t i = 0; i < columns.size(); i++) {
       // Check whether the column is used. colUsed has one bit for each of the
       // first 63 columns, and the 64th bit indicates that at least one other
       // column is used.
-      uint64_t flag;
-      if (i < 63) {
-        flag = 1LL << i;
-      } else {
-        flag = 1LL << 63;
-      }
-      if ((pIdxInfo->colUsed & flag) != 0) {
-        colsUsed.insert(std::get<0>(columns[i]));
+      auto bit = i < 63 ? i : 63U;
```

Comment:
Do we have and intention to say something in the log about index more then maximum?

Reply:
I don't think so. It's not a correctness thing, just an efficiency one, since *any* column past the 63rd being used will result in *all* columns past the 63rd being generated.

Reply:
May I ask you to define a constant for this number?